generated/nifake/nifake/_library_interpreter.py

```diff
@@ -511,9 +517,8 @@ def parameters_are_multiple_types(self, a_boolean, an_int32, an_int64, an_int_en
         an_int_enum_ctype = _visatype.ViInt16(an_int_enum.value)  # case S130
         a_float_ctype = _visatype.ViReal64(a_float)  # case S150
         a_float_enum_ctype = _visatype.ViReal64(a_float_enum.value)  # case S130
-        string_size_ctype = _visatype.ViInt32(0 if a_string is None else len(a_string))  # case S160
```

Comment:
The reference to this parameter was [removed](https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/386190), because strings with direction=in don't need a string_size, so it's no longer needed.